README.md

```diff
@@ -9,92 +9,70 @@
-  (`http://localhost:55681/v1/traces`).
+  configured to send spans to a locally running [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) over HTTP
+  (default endpoint: `http://localhost:55681/v1/traces`).
 - Unlimited default limits for [configuration options](#trace-configuration) to
   support full-fidelity traces.
 
 If you're currently using the SignalFx Tracing Library for Node and want to
 migrate to the Splunk Distribution of OpenTelemetry Node, see [Migrate from
 the SignalFx Tracing Library for JS](./MIGRATING.md).
 
-> :construction: This project is currently in **BETA**. It is **officially supported** by Splunk. However, breaking changes **MAY** be introduced.
+## Get started
 
-## Getting Started
+The following instructions assume that you're sending data to Splunk Observability Cloud using the [OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/opentelemetry.html) running on localhost. If you're running a
+different setup, refer to the [configuration options](./docs/advanced-config.md) to customize your settings.
 
-Assuming the default Splunk APM setup with [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) running on localhost. If you're running a
-different setup, refer to the [configuration options](#env-config-options) below to customize trace export endpoint
-and other behaviour.
-
-1. Install `@splunk/otel` package
+1. Install the `@splunk/otel` package:
 
 ```
 npm install @splunk/otel --save
 ```
 
-2. Install instrumentation packages
+2. Install the instrumentation packages for your library or framework:
 
 ```
 npm install @opentelemetry/instrumentation-http --save
 ```
 
-> Note: If you are using NPM 6 or older, it'll warn you about missing peer
-> dependencies. All of these dependencies are instrumentation packages and are
-> completely optional. You can install the ones you need and ignore the rest.
-> NPM 7+ supports optional peer dependencies feature and will not complain
-> about this.
-
 You can find a list of instrumentation packages supported out of the box [here](#default-instrumentation-packages).
 
-You can also install additional packages and use them as described [here](#custom-instrumentation-packages).
-
+You can also install additional packages and use them as described in [Plugins](./docs/plugins.md).
 
-3. Run node app with `-r @splunk/otel/instrument` CLI argument
+3. Run node app with the `-r @splunk/otel/instrument` CLI argument
 
 ```
 export OTEL_SERVICE_NAME=my-node-svc
 node -r @splunk/otel/instrument app.js
 ```
 
-That's it - the telemetry data is not sent to the locally running Opentelemetry Collector! You can also instrument your app with code as described [here](#instrument-with-code).
+That's it - the telemetry data is not sent to the locally running Opentelemetry Collector. You can also instrument your app with code as described [here](#instrument-with-code).
```

Comment:
```suggestion
That's it - the telemetry data is now sent to the locally running Opentelemetry Collector. You can also instrument your app with code as described [here](#instrument-with-code).
```

Reply:
```suggestion
That's it - the telemetry data is not sent to the locally running Opentelemetry Collector. You can also instrument your app programmatically as described [here](#instrument-with-code).
```
"with code" sounds childish to me. Can be easily convinced otherwise though.